roles/sft-server/vars/main.yml

```diff
@@ -0,0 +1,49 @@
+# NOTE: considered to be constants due to high precedence
+
+SFT_ARTIFACT_TARGET_PATH: '/tmp/sft-artifact.tar.gz'
+SFT_BIN_PATH: '/usr/local/bin/sftd'
+
+# NOTE: currently hardcoded in source, but will soon be configurable
+SFTD_SIGNALING_PORT: 8585
+SFTD_METRICS_PORT: 49090
+
+SFTD_INCOMING_IP: '127.0.0.1'
+
+SFTD_USER: 'sft'
+SFTD_GROUP: 'sft'
+
+# TODO: adjust calculation when/if an actual port range is defined. Also, see task 'allowing sft media ingress'
+SFTD_LIMIT_NOFILE: "{{ 65536 - 1024 }}"
+
+
+
+NGINX_CONF_PATH: '/etc/nginx/nginx.conf'
```

Comment:
A range of what you say are "constants" here, such as
 
NGINX_CONF_PATH: '/etc/nginx/nginx.conf'

are not actually constants - nginx can very well be configured to take a different path to read its config from, so it A) makes no sense to specify a variable that is already the default, and B) disallow anyone from changing the location of their nginx file by moving this variable definition into `vars/`.

Please either remove this line and its usages entirely; or move this to `defaults/` and rename to lower_case.

Reply:
Adjusted.